Monofoxe/Monofoxe.Engine/ColorExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+
+
+namespace Monofoxe.Engine
+{
+	/// <summary>
+	/// <see cref="Color"/> extensions
+	/// </summary>
+	public static class ColorExtensions
+	{
+
+		/// <summary>
+		/// Return the HsvColor equivalent of a given color 
+		/// </summary>
+		/// <param name="color">The color that will be converted</param>
+		/// <returns><see cref="HsvColor"/> equivalent of the color</returns>
+		public static HsvColor ToHsvColor(this Color color)
```

Comment:
Move all this to HsvColor. Make a constructor which accepts regular Color as a parameter. Here just do the
```
public static HsvColor ToHsvColor(this Color color) =>
  new HsvColor(color);
```